aspnetcore/signalr/configuration.md

```diff
@@ -325,6 +325,66 @@ var connection = new signalR.HubConnectionBuilder()
 
 ---
 
+### Configure stateful reconnect
+
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+
+Stateful reconnect achieves this by:
+
+* Temporarily buffering data on the server and client.
+* Acknowledging messages received (ACK-ing) by both the server and client.
+* Recognizing when a connection is returning and replaying messages that might have been sent while the connection was down.
+
+Stateful reconnect is available in ASP.NET Core 8.0 and later.
+
+Opt in to stateful reconnect at both the server hub endpoint and the client:
+
+* Update the server hub endpoint configuration to enable the `AllowStatefulReconnects` option:
+
+  ```csharp
+  app.MapHub<MyHub>("/hubName", options =>
+  {
+      options.AllowStatefulReconnects = true;
+  });
+  ```
+
+  Optionally, the maximimum buffer size in bytes allowed by the server can be set globally or for a specific hub with the `StatefulReconnectBufferSize` option:
```

Comment:
And if not set explicitly, the maximum buffer size is what? (You answer that for the client but I don't see it here for the server.)

Typo on "maximum"
```suggestion
  Optionally, the maximum buffer size in bytes allowed by the server can be set globally or for a specific hub with the `StatefulReconnectBufferSize` option:
```

Reply:
Great point, I didn't provide the default here.  Checking the source code it is 100,000 (bytes). I will add that.
    /// <summary>
    /// Gets or sets the maximum bytes to buffer per connection when using stateful reconnect.
    /// </summary>
    internal long StatefulReconnectBufferSize { get; set; } = 100_000;